profiles/signals.py

```diff
@@ -64,7 +65,12 @@ def profile_changes_to_keycloak(sender, instance, **kwargs):
 
     user_id = instance.user.uuid
 
-    user_data = _keycloak_admin_client.get_user(user_id)
+    try:
+        user_data = _keycloak_admin_client.get_user(user_id)
+    except HTTPError as err:
+        if err.response.status_code == 404:
```

Comment:
The user not being found might indicate a problem with the overall consistency which might be good to log? I do realize that at first all users have not gone through Keycloak at all and it's completely normal, but maybe later (and even earlier) we might want to follow the amount of errors here (at least in aggregate).

Reply:
Perhaps. The problem is that logging isn't well (or at all?) established in this project so it should be first figured out what exactly to log and how.

Reply:
Sure! These individual logging suggestions should still be documented somewhere so I'll save a link to this conversation.